editor-server/src/resolvers/positionMap.ts

```diff
@@ -46,19 +46,20 @@ export class EditPosMapResolver {
     @Ctx() ctx: TContext
```

Comment:
This function seems to use the old protocol, new protocol will not have dancerName in input

Reply:
Then what do we get? only frameId and dancerId?

Reply:
I've add the new protocol of position on notion's "new_protocol". You can go there and take a look at it.